docs/Versions.asciidoc

```diff
@@ -1,4 +1,4 @@
-:version:               6.0.0-rc1
+:version:               6.0.0-rc2
```

Comment:
This should not be changed here now. It needs to stay RC1 until RC2 is released.

Reply:
Ah, right. docs have a slightly different cycle. will update